Source/WebCore/platform/graphics/cocoa/GraphicsContextGLCocoa.mm

```diff
@@ -714,7 +759,11 @@ static bool needsEAGLOnMac()
         && enableExtension("GL_ANGLE_framebuffer_blit"_s)
         && enableExtension("GL_EXT_sRGB"_s)
         && enableExtension("GL_OES_EGL_image"_s)
-        && enableExtension("GL_OES_rgb8_rgba8"_s);
+        && enableExtension("GL_OES_rgb8_rgba8"_s)
+        && enableExtension("GL_NV_framebuffer_blit"_s)
+#if !PLATFORM(IOS_FAMILY_SIMULATOR)
+        && enableExtension("GL_ANGLE_variable_rasterization_rate_metal"_s);
+#endif
```

Comment:
I made a mistake here. Should be:
```
         && enableExtension("GL_EXT_sRGB"_s)
         && enableExtension("GL_OES_EGL_image"_s)
         && enableExtension("GL_OES_rgb8_rgba8"_s)
-        && enableExtension("GL_NV_framebuffer_blit"_s);
 #if !PLATFORM(IOS_FAMILY_SIMULATOR)
         && enableExtension("GL_ANGLE_variable_rasterization_rate_metal"_s)
 #endif
+        && enableExtension("GL_NV_framebuffer_blit"_s);
 }
 #endif
```

Reply:
Oh yes otherwise it won’t compile for sim